ui/src/app.js

```diff
@@ -722,6 +722,8 @@ function targetButton (element) {
 
 function touchEvent (element, callback) {
   function handler (e) {
+    // Only handle touch events on gaia
+    if (e.type === 'touchend' && cryptomatModel !== 'gaia') return
```

Comment:
I think this should be by browser version. We want touch events on all future models (Sintra basic and Sintra forte, which use upboard).